editor/src/clj/editor/gui.clj

```diff
@@ -1566,13 +1566,12 @@
   (output anim-data g/Any
           (g/fnk [costly-gui-scene-info texture]
             (let [texture-infos (:texture-infos costly-gui-scene-info)
-                  texture-info (or (get texture-infos texture)
-                                   (get texture-infos ""))]
+                  texture-info (get texture-infos texture)]
               (:anim-data texture-info))))
   (output gpu-texture TextureLifecycle (g/fnk [costly-gui-scene-info texture]
                                          (let [texture-gpu-textures (:texture-gpu-textures costly-gui-scene-info)]
                                            (or (get texture-gpu-textures texture)
-                                               (get texture-gpu-textures "")))))
+                                               @texture/white-pixel))))
```

Comment:
Here, `(get texture-gpu-textures "")` used to return the texture of the internal texture node, which was always a white pixel. Now, we use it as a fallback directly.